ceph/tox.ini

```diff
@@ -18,7 +18,8 @@ deps =
     -e../datadog_checks_base[deps]
     -rrequirements-dev.txt
 setenv =
-    CEPH_VERSION=v3.2.1-stable-3.2-mimic-centos-7
+    3: CEPH_VERSION=v3.2.1-stable-3.2-mimic-centos-7
+    15: CEPH_VERSION=v5.0.4-stable-5.0-octopus-centos-8
```

Comment:
why 15 and not 5?

Reply:
I've updated that to 5 to match the image version, but I had it as 15 due to this article https://docs.ceph.com/en/latest/releases/octopus/